test/utils/DeploymentSetup.sol

```diff
@@ -136,9 +137,9 @@ abstract contract PreMigrationSetup is Setup {
         collateralManager.addCollateral(USDT, _data);
         collateralManager.addCollateral(FRAX, _data);
         collateralManager.addCollateral(USDC, _data);
-        collateralManager.addCollateralStrategy(USDCe, address(stargateStrategy), 3000);
         collateralManager.addCollateralStrategy(USDCe, address(aaveStrategy), 4000);
-        collateralManager.updateCollateralDefaultStrategy(USDCe, address(stargateStrategy));
+        console.log(collateralManager.isValidStrategy(USDCe, address(aaveStrategy)));
```

Comment:
Can we remove the console

Reply:
Done: b54fcd192e1876669558256f2d5d2e161b8c14ef